exchanges/orderbook/orderbook_test.go

```diff
@@ -103,20 +103,20 @@ func TestVerify(t *testing.T) {
 	err = b.Verify()
 	require.ErrorIs(t, err, errPriceOutOfOrder)
 
-	b.Bids = []Tranche{{Price: 100, Amount: 1}, {Price: 100, Amount: 0}}
+	b.Bids = []Level{{Price: 100, Amount: 1}, {Price: 100, Amount: 0}}
 	err = b.Verify()
 	require.ErrorIs(t, err, errAmountInvalid)
 
-	b.Bids = []Tranche{{Price: 100, Amount: 1}, {Price: 0, Amount: 100}}
+	b.Bids = []Level{{Price: 100, Amount: 1}, {Price: 0, Amount: 100}}
 	err = b.Verify()
 	require.ErrorIs(t, err, errPriceNotSet)
 }
 
 func TestCalculateTotalBids(t *testing.T) {
 	t.Parallel()
-	base := Base{
+	base := Book{
```

Comment:
```suggestion
	b := Book{
```
There's a few of these guys left in this file